File: python/soal1/Person.py
'''
    Nama Program          : Person.py
    Anggota Kelompok      : - Ariq Hakim Ruswadi            (140810200001)
                            - Alvaro Dwi Oktaviano          (140810200041)
                            - Johanes Bagus Prasetyo        (140810200043)
                            - Rafiansyah Rasyid Wikawang    (140810200049)
    Tanggal               : 8/11/2021
'''
class Person:
    def __init__(self, nama = ""):
        self._nama = nama

    def getNama(self):
        return self._nama

    def setNama(self,nama):
        self._nama = nama
        
    def toString(self):
        return "Person: " + self._nama

File: cpp/soal1/Person.cpp
/*
    Nama Program          : Person.cpp
    Anggota Kelompok      : - Ariq Hakim Ruswadi            (140810200001)
                            - Alvaro Dwi Oktaviano          (140810200041)
                            - Johanes Bagus Prasetyo        (140810200043)
                            - Rafiansyah Rasyid Wikawang    (140810200049)
    Tanggal               : 8/11/2021
*/
#pragma once
#include <iostream>
#include <string>

using namespace std;

class Person
{
protected:
    string nama;

public:
    Person()
    {
        this->nama = "";
    }

    Person(string nama)
    {
        this->nama = nama;
    }

    string getNama()
    {
        return this->nama;
    }

    void setNama(string nama)
    {
        this->nama = nama;
    }

    string toString()
    {
        return "Person: " + this->nama;
    }
};
